src/jstree-plugins.js

```diff
@@ -23,7 +23,7 @@ define([
     "use strict";
     $.jstree.defaults.conditionalevents = {
         should_activate: function () { return true; },
-        should_move: function () { return true; },
+        //should_move: function () { return true; },
```

Comment:
Keeping this around so we don't need to figure out how to do it next time if we ever want it back. Commented to reduce performance overhead.